migrate/20240210_postgres_ha.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_enum(:ha_status, %w[none async sync])
```

Comment:
Isn't this more like "default ha type" since to switch between them is merely setting a GUC? the customer may degrade this in a session or even database wide via ALTER DATABASE or ALTER SYSTEM? Originally I thought ha_status was the catching_up/ready enum, because that, too, is a status.

Reply:
changed it with ha_type

Reply:
the commits are not in sync. `673189d822104565ba61aaf440c79e674926a422` still adds it as `ha_status`.

Reply:
protip, if you want to textually hack and slash a patch series, you can use `git format-patch`, `sed`, and `git apply` in that order.

Reply:
Fixed it truly this time.